lib/index.js

```diff
@@ -24,15 +25,21 @@ function create (options) {
 
       // Handle messages
       bot.on('message', function (message) {
-        if (!message.content.indexOf(prefix)) {
+        if (!message.content.indexOf(prefix) && message.channel.type !== 'dm') {
           // Parse the name and arguments.
-          const args = message.args = message.content.split(' ');
+          let args = message.args = message.content.split(" ");
           const name = args[0].slice(prefix.length);
-          args.shift();
+          args.shift()
+          args = args.join(' ')
+          args = spawnargs(args, {
+            removequotes: 'always'
+          })
```

Comment:
Instead of splitting and joining, we could parse the string directly then pick the name out of that.

Maybe something like this:
```js
const raw = message.content.slice(prefix.length);
const args = spawnargs(raw, { remotequotes: 'always' });
const name = args[0];
args.shift();
```